### PR TITLE
Fix code formatting.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Ran CodeFormatter on Features.csprojj

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
tagging @dotnet/roslyn-ide

File: src/EditorFeatures/VisualBasicTest/CodeActions/ExtractMethod/ExtractMethodTests.vb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
 Option Strict Off
 ' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```
Comment (Pilchie) [inline src/EditorFeatures/VisualBasicTest/CodeActions/ExtractMethod/ExtractMethodTests.vb]: Looks like a bunch of these VB files got a second copyright notice above an `Option Strict Off` line.  Do we have a quick fix that is inserting the option strict above the comment banner when it shouldn't?

**Comment by Pilchie** [comment]
:+1:

**Comment by jaredpar** [comment]
:+1: 

Thanks for running that.  I've been lax in my duties :frowning:

**Pull request merged**